tests/test_manylinux.py

```diff
@@ -225,6 +226,63 @@ def test_build_wheel_with_binary_executable(docker_container):
         python_id, ['python', '-c', 'from testpackage import runit; print(runit(1.5))']).strip()
     assert output.strip() == '2.25'
 
+@pytest.mark.parametrize('with_dependency', ['0', '1'])
+def test_build_wheel_with_highest_policy_and_image_dependencies(with_dependency, docker_container):
```

Comment:
Can you try to make the line lengths shorter to make it more consistent with the rest of the file? We don't run the linter on tests at the moment but the rest of the tests are mostly under 80 characters.

Reply:
done